code401challenges/src/main/java/code401challenges/ArrayShift.java

```diff
@@ -1,7 +1,34 @@
 package code401challenges;
 
+import java.util.Arrays;
+
 public class ArrayShift {
+    public static void main (String[] args) {
```

Comment:
this is a good start at testing, but dont forget to add some tests to your test file. also make sure your test suite is jUnit4, i noticed that you had it set up as a groovy test